### PR TITLE
Cartid as string

### PR DESCRIPTION
**Pull request by DaanKouters**
### Related issues
Forum Topic: https://forum.vuestorefront.io/t/magento-2-3-with-vue-storefront-error-placing-an-order-cart-id-should-be-a-string-but-is-integer/88/5

### Short description and why it's useful
For some reason Magento generates integer and strings as cart ids. As the Elastic Search schema for orders expects always a string i propose the cast the cart_id (cartServerToken) as a string

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [x] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/doc/Upgrade%20notes.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)
- [x] I read the [TypeScript Action Plan](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/TypeScript%20Action%20Plan.md) and adjusted my PR according to it
- [x] I read about [Vue Storefront Modules](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/api-modules/about-modules.md) and I am aware that every new feature should be a module

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.10.0-rc.1] - 2019.06.19
 
 ### Added
+- Cast cart_id as string - Order schema expects string, Magento does not generate a string as cart id in any case
```
Comment (patzick) [inline CHANGELOG.md]:
```suggestion
- Cast cart_id as string - Order schema expects string, Magento does not generate a string as cart id in any case - @DaanKouters (#3097)
```

**Comment by GabiDivante** [comment]
@patzick QA approved :)

**Comment by patzick** [comment]
@DaanKouters please in future mark checkbox which allows to edit PR, i could update changelog and merge that in :)

**Comment by DaanKouters** [comment]
> @DaanKouters please in future mark checkbox which allows to edit PR, i could update changelog and merge that in :)

Hi, ok sorry should i change anything on this PR?

**Comment by patzick** [comment]
@DaanKouters just changelog - it was my previous review above :) you have suggestion there, just commit in in

**Comment by DaanKouters** [comment]
@patzick OK, done 👍 
Also cut some slack for magento, it's not in every case that's it's wrong

**Pull request merged**